outputs.tf

```diff
@@ -1,3 +1,16 @@
 output "id" {
   value = "${null_resource.default.triggers.id}"
-}
\ No newline at end of file
+}
+
```

Comment:
The objective here is that later in an `aws_ec2_instance` resource, we can just to `tags = "${my_label.tags}"`